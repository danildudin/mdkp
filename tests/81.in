22 6
803 63 231 887 329 290 383 310 296 265 546 773 374 857 636 268 909 294 98 178 488 506 
56 121 217 203 272 364 540 328 383 821 401 353 760 221 193 396 896 450 715 117 880 210 6379
43 973 868 152 375 240 364 785 638 374 153 501 435 305 290 395 36 934 390 412 912 894 4030
845 371 506 639 611 632 334 387 399 542 937 872 249 332 982 370 675 311 57 831 537 389 1129
755 894 194 242 896 247 447 177 817 724 542 639 205 886 149 514 248 171 859 331 262 645 1230
489 952 327 684 242 543 185 228 695 460 735 356 160 586 306 563 687 156 672 450 790 498 6156
285 900 115 748 956 547 321 24 800 344 101 798 481 687 958 246 701 213 129 671 244 401 6140
