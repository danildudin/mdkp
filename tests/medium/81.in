2 1
94 85 
63 96 214
