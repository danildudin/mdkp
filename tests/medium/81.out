179
2
0 1 
